js/src/components/shipping-rate-section/shipping-rate-section.js

```diff
@@ -51,27 +63,29 @@ const ShippingRateSection = () => {
 				<Section.Card>
 					<Section.Card.Body>
 						<VerticalGapLayout size="large">
-							<AppRadioContentControl
-								{ ...inputProps }
-								label={ createInterpolateElement(
-									__(
-										'<strong>Recommended:</strong> Automatically sync my store’s shipping settings to Google.',
-										'google-listings-and-ads'
-									),
-									{
-										strong: <strong></strong>,
-									}
-								) }
-								value="automatic"
-								collapsible
-							>
-								<RadioHelperText>
-									{ __(
-										'My current settings and any future changes to my store’s shipping rates and classes will be automatically synced to Google Merchant Center.',
-										'google-listings-and-ads'
+							{ ! hideAutomatticShippingRate && (
+								<AppRadioContentControl
+									{ ...inputProps }
+									label={ createInterpolateElement(
+										__(
+											'Automatically sync my store’s shipping settings to Google.',
+											'google-listings-and-ads'
+										),
+										{
+											strong: <strong></strong>,
+										}
 									) }
```

Comment:
Since we dont use anymore <strong> tag we can omit  createInterpolateElement and just use 

```js
label={
					__(
											'Automatically sync my store’s shipping settings to Google.',
											'google-listings-and-ads'
										)
}
								
```